latest/index.bs

```diff
@@ -350,7 +350,7 @@ can be found under the "omero" key in the group-level metadata:
 }
 ```
 
-See https://docs.openmicroscopy.org/omero/5.6.1/developers/Web/WebGateway.html#imgdata
+See the [OMERO WebGateway documentation](https://docs.openmicroscopy.org/omero/5.6.1/developers/Web/WebGateway.html#imgdata)
```

Comment:
Only question is whether we should start using https://omero.readthedocs.io/en/stable/developers/Web/WebGateway.html#imgdata?